website/content/docs/connect/config-entries/proxy-defaults.mdx

```diff
@@ -12,51 +12,138 @@ description: >-
 -> **v1.8.4+:** On Kubernetes, the `ProxyDefaults` custom resource is supported in Consul versions 1.8.4+.<br />
 **v1.5.0+:** On other platforms, this config entry is supported in Consul versions 1.5.0+.
 
-The `proxy-defaults` config entry kind (`ProxyDefaults` on Kubernetes) allows for configuring global config
-defaults across all services for Connect proxy configuration. Currently, only
-one global entry is supported.
+The `proxy-defaults` config entry kind (`ProxyDefaults` on Kubernetes) allows
+for configuring global config defaults across all services for Connect proxy
+configuration. Currently, only one global entry is supported.
 
-## Sample Config Entries
+## Introduction
 
-### Default protocol
+You can customize some service registration settings for service mesh sidecar
+proxies centrally using the `proxy-defaults` configuration entry in the `kind`
+field.
+
+You can still override this centralized configuration for specific services
+with the [`service-defaults`](/docs/connect/config-entries/service-defaults)
+configuration entry `kind` or for individual proxy instances in their [sidecar
+service definitions](/docs/connect/registration/sidecar-service).
+
+## Requirements
+
+- A Consul binary
+
+## Usage
 
-Set the default protocol for all sidecar proxies:
+1. Verify that your datacenter meets the conditions specified in the [Requirements](#requirements).
+1. Apply the configuration using one of the following methods:
```

Comment:
I believe the same blob from `exported-services` needs to be adjusted like this to make sense. The usage is telling you the ways you can create it, but on the `exported-services` page it's presented like the agent config file bootstrap method is somehow step 2 instead of one of the 3 options for applying the config entry.

Reply:
A configuration entry is a structured blob of data that has to eventually end up in the state store to do its job. It has (amusingly) nothing to do with the consul agent configuration file per-se other than the config entry bootstrap special agent configuration section (which folks should practically only be using for demos and toy environments).

The primary way to get a configuration entry into the state store is via an API call to the `v1/config` endpoint. This is the mechanism used by both `consul config write` and a k8s CRD under the covers. These all fall into the same bucket of usefulness (as would using `terraform` to do the same).

Since all of the primary and related ways to get configuration entries into the state store require a server to be up and a leader to be elected for bootstrapping small environments or demos it can be annoying to deterministically wait until things _can_ accept a write before doing your initial configuration entries writes.

For those scenarios we have the `config_entries { bootstrap = [ ... ] }` https://www.consul.io/docs/agent/options#config_entries_bootstrap **agent** configuration section that only applies to servers. This is applied every time _this_ agent becomes an elected leader and the data is re-updated into the database. For scenarios where you have more than one server (like production) it should not be used because if you have slightly different things in the configuration files for those different server agents simply switching which one is leader would cause the configuration entries to flip flop back and forth.

Reply:
To clarify from the exported-services page:
```
1. Verify that your datacenter meets the conditions specified in the Requirements.
2. Specify the exported-services configuration in the agent configuration file (see config_entries) as described in Configuration.
3. Apply the configuration using one of the following methods:
  - Kubernetes CRD: Refer to the Custom Resource Definitions documentation for details.
  - Issue the consul config write command: Refer to the Consul Config Write documentation for details.
```

(2) does not mean "come up with the struct you want to save" which would go quite well as a prereq to (3) "save the thing you just came up with"

(2) is "save the thing one way" and (3) is "save the thing two other ways"